README.md

```diff
@@ -136,41 +136,45 @@ To deploy your local workbench API code to a given AppEngine project, in the api
 directory run:
 
 ```
-./project.rb deploy-api --project PROJECT --account ACCOUNT@pmi-ops.org --version VERSION [--promote]
+./project.rb deploy --project PROJECT --version VERSION --[no-]promote
```

Comment:
Do we have to specify promote or no-promote?

Reply:
Yes. I'm thinking it should be required. My thinking is that the traffic-switching behavior is not obvious; I might expect that if I'm calling a command named "deploy", my changes will go live. Better to make the caller aware of the --promote vs --no-promote decision. Even for someone who is familiar with GAE deploys, I don't think that --no-promote as a default is really saving much typing; --promote is more typical (and we probably shouldn't make that be the default).

Reply:
Sounds good.